pyproject.toml

```diff
@@ -318,10 +318,34 @@ select = [
 
 ]
 respect-gitignore = true
-ignore = ["E501", "PGH003", "RET504", "RET505", "RET506", "RET507", "RET508"]
+ignore = [
+  "E501",
+  "PGH003",
+  "RET504",
+  "RET505",
+  "RET506",
+  "RET507",
+  "RET508",
+  "D100", # public module
+  "D101", # public class
+  "D102", # public method
+  "D103", # public function
+  "D105", # magic methods
+  "D107", # init
+  "D202", # blank lines after function docstring
+  "D203", # blank line before class docstring
+  "D205", # blank line between summary line and description
+  "D213", # Multi-line docstring summary should start at the second line
+  "D402", # First line should not be the function's signature
+  "D417", # Missing argument descriptions
```

Comment:
I assume we can incrementally remove these skipped rules.

Reply:
Perhaps, but there a ton of warnings, so it'll quite some time :)